pages/community/index.js

```diff
@@ -440,43 +493,20 @@ function CommunityIndexPage() {
         </div>
         <div className="mt-10 md:flex justify-between w-full h-full-screen md:h-130">
           <div className="w-full md:w-3/4 mr-5 rounded-lg relative shadow-xl bg-tutorialCover bg-cover h-half-screen md:h-full bg-center">
-            <div className="w-full h-full bg-primary-500 opacity-25 absolute rounded-lg" />
-            <div className="w-full h-full flex flex-col justify-end absolute">
-              <div className="bg-primary-400 text-left rounded-b-lg p-5 ">
-                <Heading
-                  level="h2"
-                  typeStyle="heading-lg"
-                  className="text-white"
-                >
-                  Browse our Tutorials
-                </Heading>
-                <Heading
-                  level="h2"
-                  typeStyle="body-lg"
-                  textColor="text-gray-900"
-                  className="text-slate-500 text-sm"
-                >
-                  We have thousands of tutorials covering a wide range of
-                  topics. Jump into our catalog!
-                </Heading>
-              </div>
-            </div>
-          </div>
-          <div className="h-half-screen md:h-full w-full md:w-2/4 md:flex md:flex-col md:justify-around mt-10 md:mt-0">
-            <div className="h-4/6 md:h-beforeHalf rounded-lg shadow-lg bg-usecases bg-cover bg-center relative">
-              <div className="w-full h-full bg-pink-500 opacity-25 absolute rounded-lg" />
+            <a href="/docs/tutorials">
```

Comment:
This should be a `Link` tag from `next/link` instead of `a` tag.

Reply:
Lol. I prefer using this method instead of next/link for external link. Is there a specific reason why you think i should change it?

Reply:
Yeah, because `Link` tag provides better routing facility than `a` tag in Next js. For more details, you can visit: 

https://nextjs.org/docs/routing/introduction#linking-between-pages

Reply:
Oh! sorry i thought you were talking about linking to a different domain entirely